lib/vagrant-packer-plugin/utils/packer_interface.rb

```diff
@@ -92,22 +91,18 @@ def parametrize(packer, type, components)
     		config.send("#{key}", value) if key != "type"
     	end
 
-
 			if type == "Provisioner" and component['type'] == "chef-solo"
     		config.required = ["type"]
-    		config.run_list  packer.variables["run_list_item"].split(",")
+    		config.run_list  packer.variables["run_list"].split(",")
```

Comment:
Does this mean that now the variable run_list_item needs to be renamed to run_list ?
https://github.com/Alfresco/alfresco-spk/commit/7edf61c7af74b57e25f6ace143ac50ba7f6054fe (look at instance-templates/allinone-community.json )